### PR TITLE
Enable `template-curly-spacing`

### PR DESCRIPTION
**Pull request by shreyasminocha**
Since we're apparently fans of spacing our brackets out, here we go.

### CODE REVIEW

File: language/es6.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "no-new-require": "error",
     "no-useless-computed-key": "error",
     "one-var": "off",
-    "vars-on-top": "off"
+    "vars-on-top": "off",
+    "template-curly-spacing": "error"
```
Comment (Volker-E) [inline language/es6.json]: Rules are ordered alphabetically here.

**Comment by edg2s** [comment]
The default option is "never", I assume you meant to use "always"?

https://eslint.org/docs/rules/template-curly-spacing#always

**Comment by edg2s** [comment]
Adding spaces would seem consistent with the rest of the ruleset, however I can see an argument for not adding non-meaningful whitespace to a string where whitespace is meaningful:

e.g. it may be easier to see the spacing is correct in the first example:
`The ${speed} brown fox`
`The ${ speed } brown fox`

**Comment by Krinkle** [comment]
Superseded by #168, in which we went for not having spaces around variable interpolation.

**Pull request closed without merging**